modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -417,8 +415,8 @@ include::devguide:example$java/StartUsingCapella.java[tag=replace-named,indent=0
 ----
 ////
 
-CAUTION: When you replace a document, it's usually good practice to use xref:howtos:kv-operations.adoc#optimistic-locking[optimistic locking].
-Otherwise, changes might get lost if two people change the same document at the same time.
+CAUTION: While replacing a document, it is a good practice to use xref:howtos:kv-operations.adoc#optimistic-locking[optimistic locking].
```

Comment:
I'm all for opinionated recommendations as guard rails -
in this case I need to think through some of the situations where even optimistic locking is not the best route,
and see what we should say and where before we recommend strongly.

Would value any thoughts and arguments for or agin.